### ISSUE TITLE
Slider generating junks

### ISSUE THREAD

**Issue by adamos42** [zurb/foundation-sites-6#213]
The built in slider generating junks, because the timer does not wait the next frame to call the animation function, also timers should use in workers for permormace, because mobile devices rendering can easily be blocked by timers in main thread. Use WebWorker for timing and CSS transform translate for positioning because thats not need recalculate the layer, and the slowness on mobie devices should solved.
http://foundation.zurb.com/sites-6-docs/slider.html

**Comment by zurbrandon** [comment]
@zurbchris can take a look at this, I know he has recent;y went back too look at some of this performance implications

**Comment by adamos42** [comment]
I've made a little test after I talked with @zurbchris, turns out the slider request an AnimationFrame from the mousemove witch is right, but somehow it's not cancelled if an other mousemove event fired. In my test I've solved that: http://a42server.net/foundation/slider.html and an other interesting thing happend: it can fire a mousemove event when the animationFrame is running so I made an other requestAnimationFrame when the current is done, anything after the first frame should calculated next time in next frame. Also I changed the position left modifier and changed to css translate, because with translate the browser does not need repaint the layer.
